CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ endif()
 if(USE_ONNX)
   message(STATUS "Enabling experimental ONNX support")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_ONNX")
+  # updated sentencepiece likely requires this to find protobuf by itself
```

Comment:
The comment says "likely". Should we leave a TODO here to verify this?

Reply:
Yes, that what essentially meant to be a TODO, will update.